modules/assim.sequential/R/Analysis_sda_block.R

```diff
@@ -174,8 +174,8 @@ build.block.xy <- function(settings, block.list.all, X, obs.mean, obs.cov, t) {
       #name matching between observation names and state variable names.
       f.2.y.ind <- obs.mean[[t]] %>% 
         purrr::map(\(x)which(var.names %in% names(x))) %>% 
-        base::unlist %>% 
-        base::unique
+        unlist %>% 
```

Comment:
did you have to remove the namespace here? There's a chance GH Actions is going to complain

Reply:
I haven't yet found a case where R complains about removing `base::` -- the builtins really are special that way